feather/common/src/game.rs

```diff
@@ -229,6 +229,12 @@ impl Game {
     pub fn break_block(&mut self, pos: BlockPosition) -> bool {
         self.set_block(pos, BlockId::air())
     }
+
+    pub fn reset_player(&self, player: EntityRef) {
```

Comment:
I think just completely ignoring a failure here is not ideal, maybe return an error of some kind upon failure.